woocommerce/woocommerce
# Add e2e test for Merchant > Pages > Can create a new page

Create e2e test for Critical Flow: Merchant > Pages > Can create a new page

### Submission Review Guidelines:



### Changes proposed in this Pull Request:

This PR adds e2e-pw test for Critical flow: `Merchant > Pages > Can create page`.



### How to test the changes in this Pull Request:

1. Make sure you meet the [Pre-requisites](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/tests/e2e-pw#pre-requisites)
2. Run `pnpm test:e2e-pw .tests/e2e-pw/tests/merchant/create-page.spec.js`
3. A new page [_Title: Page-#EPOCHTIME_] should be created first, and then deleted via API
4. The page should be deleted from trash via WP API
